src/form/form.tsx

```diff
@@ -308,6 +308,9 @@ class Form<MODEL extends object> extends React.Component<Props<MODEL>, State<MOD
         return getDeclaredFields(this.props.children)
             .map((field: Field<any, MODEL>) => field.id)
             .map(this.getFieldViolations)
+            .filter((violations: Array<ConstraintViolation>) =>
+                violations.every((violation) => violation.severity === ViolationSeverity.ERROR)
```

Comment:
Doesn't that return true if you have warning and error for a particular field?

Reply:
It will return `false` If you have for e.g `[INFO, ERROR]`.

Reply:
Clarified in person